### ISSUE TITLE
Pagination using ColumnLayout

### ISSUE THREAD

**Issue by iashu90** [viht0ri/ColumnLayout#2]
Is there any way to do pagination with this with the long text and images? 
Using ViewPager?

And in my case number of pages exist one page.

**Comment by viht0ri** [comment]
Yeah you should be able to calculate the columns for each page by starting from the end of the previous page.

Did not really get what you mean with "And in my case number of pages exist one page."

**Comment by iashu90** [comment]
When I pass the SpannedText to ColumnLayout , when text overflows , its getting normal text( without HTML) , so from second page , cant able to to SpannedText with HTMl tags
